cloudwatch/ecr-source-event.json

```diff
@@ -0,0 +1,24 @@
+{
+  "source": [
+    "aws.ecr"
+  ],
+  "detail-type": [
+    "AWS API Call via CloudTrail"
+  ],
+  "detail": {
+    "eventSource": [
+      "ecr.amazonaws.com"
+    ],
+    "eventName": [
+      "PutImage"
+    ],
+    "requestParameters": {
+      "repositoryName": [
+        "${ecr_repository_name}"
+      ],
+      "imageTag": [
+        "latest"
```

Comment:
Does it have to be the generic `latest` tag? Could we use the actual commit shasum instead?

I am worried what would happen if multiple concurrent pushes occurred and there would be more than a single pending deployment running.

Reply:
I think it has to be `latest` or any specific tag. If the tag is unset it falls back to `latest` anyway... So we're kinda limited to push the image tagged as `latest` in order to trigger CodePipeline 👎 but the actual pull from Fargate is related to the shasum of the image, so we avoid that "same latest different shasum" problem of Docker

Reply:
Ok, I am not sure I fully grok the behaviour but if that problem is solved elsewhere I am okay with this! 👍 💯